Extension/src/LanguageServer/client.ts

```diff
@@ -925,7 +930,7 @@ export class DefaultClient implements Client {
     public get AdditionalEnvironment(): { [key: string]: string | string[] } {
         return {
             workspaceFolderBasename: this.Name,
-            workspaceStorage: this.storagePath,
+            workspaceStorage: this.legacyStoragePath,
```

Comment:
This value is used by `resolveVariables` any time the user specifies the `workspaceStorage` variable.  Shouldn't the new storage location be used here, and the legacy storage path never used anymore (except to delete an old database) ?

Reply:
No, it's fine -- the workspaceStorage is still the workspaceStorage/legacyStoragePath -- the only thing that is changing is that we're not storing the database at that spot -- other stuff might still be stored there.

Reply:
...if we wanted to add a databaseStoragePath or something that would be a new variable.

Reply:
My assumption is that users are not using this variable to refer to our default database location.

Reply:
If there is a reason to store our database in a different storage location, it's unclear to me why the same reasoning would not apply to anything else that similarly needs storage.

Reply:
Because it's large file that is a cache that is potentially deletable (going to XDG_CACHE_HOME on Linux), where-as other workspace storage is supposed to be small and may not be deletable.

Reply:
Should we use a better name for the variable, then, since it's still actively used and not a 'legacy' holdover?

Reply:
Better name for what variable?

Reply:
We figured out the renaming...